Added Python to highlite module.

xbello opened the pull request:
* Copy-pasted the Nim Tokenizer to remove the non-existent multiline comment
operator, and added a 'keywords' parameter (like in cLikeNextToken).

* Added the keywords for Python

xbello committed:
Added Python to highlite module.

* Copy-pasted the Nim Tokenizer to remove the non-existent multiline comment
operator, and added a 'keywords' parameter (like in cLikeNextToken).

* Added the keywords for Python

Araq commented:
> Copy-pasted the Nim Tokenizer to remove the non-existent multiline comment
operator, and added a 'keywords' parameter (like in cLikeNextToken).

Instead patch the Nim Tokenizer and disable the multiline comment for ``g.lang == langPython`` (if ``g.lang`` doesn't yet exist, introduce it).

Araq commented on lib/packages/docutils/highlite.nim:
```diff
@@ -886,6 +1042,16 @@ proc yamlNextToken(g: var GeneralTokenizer) =
+proc pythonNextToken(g: var GeneralTokenizer) =
+  const
+    keywords: array[0..34, string] = [
+      "False", "None", "True", "and", "as", "assert", "async", "await",
```
Iirc, False, None and True are not keywords.

xbello committed:
Refactored the 'pythonLikeNextToken' into 'nimNextToken':

- `lang` property added to GeneralTokenizer object, is set in `getNextToken`.
- `nimNextToken` accepts `keywords` parameter, used for languages different
  from Nim.
- Multiline comment available only for `langNim`.

xbello reviewed (commented):
Had to made a couple of additional changes to pass the tests (tests/stdlib/trstgen.nim).

Araq commented:
Excellent work!
